synapseclient/__main__.py

```diff
@@ -547,7 +544,11 @@ def build_parser():
     parser.add_argument('-c', '--configPath', dest='configPath', default=synapseclient.client.CONFIG_FILE,
                         help='Path to configuration file used to connect to Synapse [default: %(default)s]')
 
-    parser.add_argument('--debug', dest='debug', action='store_true')
+    parser.add_argument('--debug', dest='debug', action='store_true', help='Set mode to debug mode, default is False')
```

Comment:
"Set to debug mode, additional output and error messages are printed to the console"